core/templates/dev/head/pages/library-page/search-bar/search-bar.directive.html

```diff
@@ -16,15 +16,19 @@
   </form>
 </div>
 
+
```

Comment:
Please remove these extra blank lines. Only one blank line should be left.

Reply:
done!